src/views/NoteView.vue

```diff
@@ -1,5 +1,123 @@
 <template>
-  <h1>Note View</h1>
+  <div class="note-view">
+    <div class="header-controls">
+      <div class="note-info">
+        <div class="note-channel">
+          <span>channel:</span>
+          <span class="note-channel">{{ note.channel }}</span>
+        </div>
+        <div class="note-tag">
+          <span>tags: </span>
+          <span class="note-tag">{{ note.tag }}</span>
+        </div>
+      </div>
+      <div class="note-actions">
+        <button class="note-action-button">ホームへ戻る</button>
+        <button class="note-action-button">編集</button>
+      </div>
+    </div>
+    <h1 class="note-title">{{ note.title }}</h1>
+    <div class="note-body">
+      <p>{{ note.summary }}</p>
+    </div>
+    <div class="note-body" v-html="renderedMarkdown"></div>
+
+
+  </div>
 </template>
 
-<script setup lang="ts"></script>
+<script setup lang="ts">
+import type { NoteSummary } from '@/types/api';
+import { ref, computed } from 'vue';
+import { marked } from 'marked';
+import DOMPurify from 'dompurify';
+
+const renderedMarkdown = computed(() => {
+  if (!note.value.summary) {
+    return '';
+  }
+  // markedでMarkdownをHTMLに変換し、DOMPurifyでサニタイズする
+  const rawHtml = marked(note.value.summary) as string;
+  return DOMPurify.sanitize(rawHtml);
+});
+
+
+
+const note = ref<NoteSummary>({
+  id: '',
+  permission: 'private',
+  channel: '',
+  tag: '',
+  title: '',
+  summary: '',
+  createdAt: 0,
+  updatedAt: 0
+});
+
+
+
+
+
+
+</script>
+
+<style scoped>
+.note-view {
+  max-width: 1200px;
+  margin: auto;
+
+}
+
+.header-controls {
+  display: flex;
+  /* Flexboxコンテナにする */
+  justify-content: space-between;
+  /* 左右の要素を両端に配置 */
+  align-items: center;
+  /* 垂直方向の中央に揃える */
+  margin-bottom: 20px;
+  /* タイトルとの間に少し余白 */
+}
+
+.note-info {
+  display: flex;
+  /* channelとtagsもFlexboxで横並びにする */
+  align-items: center;
+  /* 垂直方向の中央に揃える */
+  gap: 20px;
+  /* channelとtagsの間のスペース */
+}
+
+.note-channel {
+  font-size: 16px;
+}
+
+.note-tag {
+  font-size: 16px
+}
+
+
+
+
+.note-actions {
+  display: flex;
+  gap: 10px;
+}
+
+.note-action-button {
+  font-size: 13px;
+  width: 130px;
+  height: 32px;
+  margin-top: 20px;
+  background-color: #58b5828a;
+  border: 2px solid #6edfa115;
+  border-color: #6edfa115;
+  border-radius: 3px;
+}
```

Comment:
width/height 指定ではなく padding で指定することで、テキストの長さを基準として自動で均等な余白が確保できます！もし、多言語対応だとか、複数ボタンがあるだとかするときに便利なので padding での指定を推奨します！

```suggestion
.note-action-button {
  font-size: 0.9px;
  padding: 0.6rem 1.5rem;
  background-color: #58b5828a;
  border: 2px solid #6edfa115;
  border-color: #6edfa115;
  border-radius: 3px;
}
```